src/main/kotlin/dev/stalla/model/googleplay/GoogleplayCategory.kt

```diff
@@ -1,42 +1,70 @@
 package dev.stalla.model.googleplay
 
-import dev.stalla.builder.GoogleplayCategoryBuilder
-import dev.stalla.builder.validating.ValidatingGoogleplayCategoryBuilder
-import dev.stalla.model.BuilderFactory
-
 /**
  * An [Google Play `<category>` tag][https://support.google.com/googleplay/podcasts/answer/6260341#spt].
  *
- * @param name The name of the category.
+ * @param type The string representation of the category.
  */
-public sealed class GoogleplayCategory(public open val name: String) {
+public enum class GoogleplayCategory(public val type: String) {
 
-    public companion object Factory : BuilderFactory<GoogleplayCategory, GoogleplayCategoryBuilder> {
+    /** Category type for _Arts_ */
+    Arts("Arts"),
 
-        /** Returns a builder implementation for building [GoogleplayCategory] model instances. */
-        @JvmStatic
-        override fun builder(): GoogleplayCategoryBuilder = ValidatingGoogleplayCategoryBuilder()
-    }
+    /** Category type for _Business_ */
+    Business("Business"),
```

Comment:
Note upper camel casing despite being an enum. Consistency with `ItunesCategory`

Reply:
As mentioned earlier, I think we should aim for consistency with this being an enum, more than with other models (which aren't backed by an enum). I think it makes it more usable from Java as well, since Java folks expect enums to have screaming snake case members...

Reply:
That said, if you feel strongly for camel casing names, I won't stop you (although I prefer to shout enums 🤣 ), I can live with it :)

Reply:
I don't have any special favourites, I just don't want to violate conventions. Can rename to shouty names.

Reply:
I find the notion of shouty names thoroughly entertaining. I don't know why I never considered it such. I always thought of uppercase enum names as meaning "authoritative, set in stone". But, can't unsee the shouty snakes now! 🤣

Reply:
I agree, the uppercase names always did underline their immutability. But I've learned to appreciate non shouty names from Scala a few years ago too.